frontend/packages/ux-editor/src/components/config/componentSpecificContent/FileUpload/FileUploadComponent.tsx

```diff
@@ -111,20 +110,12 @@ export const FileUploadComponent = ({
           propertyPath={`${component.propertyPath}/properties/displayMode`}
         >
           {() => (
-            <LegacyRadioGroup
-              items={[
-                {
-                  label: t('ux_editor.modal_properties_file_upload_simple'),
-                  value: 'simple',
-                },
-                {
-                  label: t('ux_editor.modal_properties_file_upload_list'),
-                  value: 'list',
-                },
-              ]}
-              name={`${component.id}-display-mode`}
-              variant='horizontal'
-            />
+            <Radio.Group name={`${component.id}-display-mode`} inline={true}>
+              {Children.toArray([
+                <Radio value='simple'>{t('ux_editor.modal_properties_file_upload_simple')}</Radio>,
+                <Radio value='list'>{t('ux_editor.modal_properties_file_upload_list')}</Radio>,
+              ])}
```

Comment:
Can this be simplified by removing the Children.ToArray? 😊 I do not understand why we need it. Maybe I'm missing something? 😊

```suggestion
                <Radio value='simple'>{t('ux_editor.modal_properties_file_upload_simple')}</Radio>,
                <Radio value='list'>{t('ux_editor.modal_properties_file_upload_list')}</Radio>,
```